webnative-filecoin/src/wallet/index.ts

```diff
@@ -71,8 +72,8 @@ export default class Wallet {
     return this.providerBalance
   }
 
-  async fundProvider(amount: number):  Promise<Receipt> {
-    if(amount > this.balance) {
+  async fundProvider(amount: number): Promise<Receipt> {
```

Comment:
wanna just switch out this method to call `send` with the provider's address?

that's how it'll work in the actual implementaiton as well :ok_hand:

Reply:
Yep, sounds good. :+1: 

Refactored that and bringing this in. Thanks for the review. :pray: